packages/website/src/content/docs/reference/configuration.md

```diff
@@ -43,7 +43,7 @@ Set this option if you have the sidecar running on another URL than the default
 
 ```ts
 init({
-  sidecarUrl: 'http://localhost:8969/stream',
+  sidecarUrl: 'http://localhost:8969/sidecar',
```

Comment:
Did you mean
```suggestion
  sidecarUrl: 'http://localhost:8969/',
```

Reply:
it is a mistake, this should not be changed.